src/devices/bus/ti99/gromport/cartridges.cpp

```diff
@@ -63,21 +60,21 @@ enum
 	PCB_PAGED7
 };
 
-static const pcb_type pcbdefs[] =
-{
-	{ PCB_STANDARD, "standard" },
-	{ PCB_PAGED12K, "paged12k" },
-	{ PCB_PAGED16K, "paged" },
-	{ PCB_MINIMEM, "minimem" },
-	{ PCB_SUPER, "super" },
-	{ PCB_MBX, "mbx" },
-	{ PCB_PAGED379I, "paged379i" },
-	{ PCB_PAGED378, "paged378" },
-	{ PCB_PAGED377, "paged377" },
-	{ PCB_PAGEDCRU, "pagedcru" },
-	{ PCB_GROMEMU, "gromemu" },
-	{ PCB_PAGED7, "paged7" },
-	{ 0, nullptr}
+static const char *pcbdefs[] =
+{
+	"standard",		// PCB_STANDARD
+	"paged12k",		// PCB_PAGED12K
+	"paged",		// PCB_PAGED16K
+	"minimem",		// PCB_MINIMEM
+	"super",		// PCB_SUPER
+	"mbx",			// PCB_MBX
+	"paged379i",	// PCB_PAGED379I
+	"paged378",		// PCB_PAGED378
+	"paged377",		// PCB_PAGED377
+	"pagedcru",		// PCB_PAGEDCRU
+	"gromemu",		// PCB_GROMEMU
+	"paged7",		// PCB_PAGED7
+	nullptr
 };
```

Comment:
This is why I do `const` the way I do – you’ve created an array of non-`const` pointers to `const` character arrays.  It should be `static char const *const pcbdefs[] =` – you’re less likely to make this error with trailing `const`.